CRM/Upgrade/Incremental/php/FiveSixty.php

```diff
@@ -43,4 +44,21 @@ public static function addScheduledReminderSmartySetting(): bool {
     return TRUE;
   }
 
+  public static function addJobLogForeignKey() {
+    // Set job_id = NULL for any that don't have matching jobs (ie. job was deleted).
```

Comment:
I always think a comment in the first line of a function is a pointer it should have a docblock. This should really have a return hint.

Neither are blocking

Reply:
That comment is specific to the first two lines and not the whole function

Reply:
@mattwire yeah - I would have put it in a doc block - but as I say, non blocking